### DISCUSSION TITLE
postgres to bigquey streaming

### DISCUSSION THREAD

**Discussion by madalina2311** [redpanda-data/connect#1321] [Q&A] [answered]
Hi . I am looking for an option to do postgres to bigquery in a continuous way. 
At the moment i am using sql_select  and it works as expected - once the rows from the query are exhausted this input shuts down, allowing the pipeline to gracefully terminate. I was wondering if there is a way to have the pipeline running and catch new events that come to the postgres db so i can send them to bigquery. Alternatively, is there a cron or a timer that i can set to trigger checks and import new data? Thanks

**Comment by mihaitodor** [comment, marked as answer]
You can try using the [`generate`](https://www.benthos.dev/docs/components/inputs/generate) input in combination with the [`sql_select`](https://www.benthos.dev/docs/components/processors/sql_select) processor. Please let me know if you need more info.

**Reply by madalina2311** [reply]
Thanks. Everything worked as expected.
